examples/docker/00-hello-world/client.js

```diff
@@ -0,0 +1,24 @@
+const Queue = require('bee-queue');
+
+// Core client logic, run one job and then exit
+const clientProcess = (queue) => {
+  const job = queue.createJob({name: 'World'});
+  job.on('succeeded', (result) => {
+    console.log();
+    console.log(result.greeting);
+    console.log();
+
+    process.exit(0);
```

Comment:
I usually prefer to let the event loop empty vs forcibly exiting. What do you think?

maybe with some error handling:
```suggestion
    queue.close();
```

Reply:
Duh, thank you!